robowflex_tesseract/src/trajopt_planner.cpp

```
@@ -406,6 +418,14 @@ TrajOptPlanner::PlannerResult TrajOptPlanner::plan(const SceneConstPtr &scene, c
     return PlannerResult(false, false);
 }
 
+TrajOptPlanner::PlannerResult TrajOptPlanner::plan(const SceneConstPtr &scene,
+                                                   const robot_state::RobotStatePtr &start_state)
+{
+    RBX_ERROR("You need to implement virtual method TrajOptPlanner::plan() in your derived class");
```

probably better to throw an exception here rather than just a print